docs/source/plugin/advanced.rst

```diff
@@ -15,6 +15,60 @@ If something is not in here, feel free to ask about it on our IRC channel, or
 maybe open an issue with the solution if you devise one yourself.
 
 
+Restricting commands to certain channels
+========================================
+
+Allowing games, for example, to be run only in specific channels is a
+relatively common request, but a difficult feature to support directly in
+Sopel's plugin API. Fortunately it is fairly trivial to build a custom
+decorator function that handles this in a configurable way.
+
+Here is a sample plugin that defines such a custom decorator, along with the
+scaffolding needed to pull a list of channels from the bot's settings::
```

Comment:
Nit: a nudge to the anaomy document would be helpful for a reader who may not be familiar but ends up finding this example because they went looking for it

```suggestion
scaffolding needed to pull a list of channels from the :ref:`bot's settings <plugin-anatomy-config>`:
```

Reply:
This will probably need to be split in two lines with that suggestion. :grin:

Reply:
Yeah.

```suggestion
scaffolding needed to pull a list of channels from the bot's
:ref:`settings <plugin-anatomy-config>`::
```

(I _think_ the `::` is required still, but I need to test-render in Sphinx before pushing a correction commit.)

Reply:
Have tested, played with wording/which parts of the sentence to link, and arrived at a happy place.